CMakeLists.txt

```diff
@@ -36,3 +39,29 @@ endif()
 add_definitions(-D_GNU_SOURCE)
 
 install(TARGETS iotools DESTINATION sbin)
+
+# Install symlinks á la `iotools --make-links`, but in a cross-build
+# compatible manner.
+if (INSTALL_LINKS)
```

Comment:
Cool, interesting trick!
The question here is how is it portable? And what if `readelf` is not available (may this be the case)?